sashimi-webapp/src/components/editor-viewer/Viewer.vue

```diff
@@ -6,7 +6,7 @@
 </template>
 
 <script>
-  import wrapper from '../../logic/wrapper';
+  import wrapper from '../../logic/documentPackager';
```

Comment:
@jiayingy Should have used

`import wrapper from 'src/logic/...'` next time

Since our webpack is configured to resolve `src` automatically